redwood.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Priyanshu Jangid",
+			"oem": "Poco",
+			"device": "Poco X5 Pro 5G",
+			"filename": "crDroidAndroid-15.0-20241107-redwood-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/redwood/11.x/crDroidAndroid-15.0-20241107-redwood-v11.0-BETA.zip/download",
+			"timestamp": 1730996903,
+			"md5": "5c890b8e2f78d75ad6bb4e5883d2fdca",
+			"sha256": "5239c75bf990c24cf13f198704892a0d0be36cd69fd5867124c46eddc06ed4a4",
+			"size": 1606222282,
+			"version": "11.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-15-0-redwood-crdroid-v11-0-official.4672931/",
```

Comment:
Remove telegram links on forum and use new support page url
Mandatory to comply with XDA rules